website/pages/docs/references/eslint.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: ESLint Plugin
+description: ESLint plugin for Panda CSS 
+---
+
+# ESLint Plugin
+
+ESLint plugin for Panda CSS: `@pandacss/eslint-plugin`.
+
+## Installation
+
+<Tabs items={['pnpm', 'npm', 'yarn', 'bun']} variant="code">
+  <Tab>
+    ```bash
+    pnpm add -D @pandacss/eslint-plugin
+    ```
+  </Tab>
+  <Tab>
+    ```bash
+    npm install -D @pandacss/eslint-plugin
+    ```
+  </Tab>
+  <Tab>
+    ```bash
+    yarn add -D @pandacss/eslint-plugin
+    ```
+  </Tab>
+  <Tab>
+    ```bash
+    bun add -D @pandacss/eslint-plugin
+    ```
+  </Tab>
+</Tabs>
+
+## Rules
+
+### `@pandacss/eslint-plugin/file-not-included`
+
+Prevent panda usage in files not convered by panda `include` config.
+
+### `@pandacss/eslint-plugin/no-debug`
+
+Prevent shipping the debug attribute to production.
+
+### `@pandacss/eslint-plugin/no-shorthand-prop`
+
+Discourage the use of shorthand properties. Prefer to use longhand css property instead.
+
+### `@pandacss/eslint-plugin/no-unsafe-token-fn-usage`
+
+Ensure user does not use token fn in places they could just use the raw design token.
+
+### `@pandacss/eslint-plugin/prefer-atomic-properties`
+
+Prefer atomic properties over shorthand properties.
+
+## Settings
+
+### `configPath`
+
+You can tell `eslint` to use a custom panda config file by setting the `configPath` option in your `.eslintrc.js` file.
+
+By default we find the nearest panda config to the linted file.
+
+```json
+{
+  "plugins": [
+    "@pandacss/eslint-plugin"
+  ],
+  "settings": {
+    "pandacss/configPath": "/ABSOLUTE-PATH-TO/panda.config.js"
+  }
+}
+```
```

Comment:
[Suggestion] Have you considered also adding a tab for the [new flat config format](https://eslint.org/docs/latest/use/configure/configuration-files-new)? We don't yet officially support it in typescript-eslint (long story) but a lot of consumers -including folks who newly set up their configs starting next month or so onward- will be defaulting to it.

Reply:
@JoshuaKGoldberg I thought of it, but I'm not sure it's setup to work with flat configs yet. Also yet to add `configs` i.e. recommended.